src/test/unit/com/hltech/pact/gen/domain/client/feign/sample/MediaTypeFeignClient.java

```diff
@@ -0,0 +1,30 @@
+package com.hltech.pact.gen.domain.client.feign.sample;
+
+import com.hltech.pact.gen.domain.client.feign.InteractionInfo;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@FeignClient("SpecProvider")
+public interface MediaTypeFeignClient {
+
+    @PostMapping(path = "/", consumes = {
+        MediaType.APPLICATION_JSON_UTF8_VALUE,
+        MediaType.APPLICATION_PDF_VALUE}, produces = {
```

Comment:
move `produces` to new line

Reply:
I think readability suffers as indent must be like in the line above.